Makefile

```diff
@@ -291,7 +291,7 @@ distclean: clean assetclean
 	$(MAKE) -C tools distclean
 
 venv:
-	test -d $(VENV) || python3 -m venv $(VENV)
+	test -d $(VENV) || python3 -m venv $(VENV) || { rm -rf $(VENV); false; }
```

Comment:
a comment is probably warranted since not everyone cares about shell syntax yet would like to know what this does

```mk
# Create the virtual environment if it doesn't exist.
# Delete the virtual environment folder if creation fails.
	test -d $(VENV) || python3 -m venv $(VENV) || { rm -rf $(VENV); false; }
```

(note: comment should be unindented otherwise make will execute it as a command (and it'll be a shell comment that shows in stdout))